anesthetic/plot.py

```diff
@@ -323,11 +323,29 @@ def kde_plot_1d(ax, data, *args, **kwargs):
     xmax = kwargs.pop('xmax', None)
     weights = kwargs.pop('weights', None)
     ncompress = kwargs.pop('ncompress', 1000)
+    percentile = kwargs.pop('percentile', '5sigma')
+    if isinstance(percentile, str):
+        sigmas = {'1sigma': 0.682689492137086,
+                  '2sigma': 0.954499736103642,
+                  '3sigma': 0.997300203936740,
+                  '4sigma': 0.999936657516334,
+                  '5sigma': 0.999999426696856}
+        percentile = sigmas[percentile]
+    if isinstance(percentile, float) or isinstance((percentile, int)):
+        if percentile < 0.5:
+            percentile = 1 - percentile
+        percentile = (1-percentile, percentile)
+        print(percentile)
```

Comment:
Cool! I would default to 3 sigma, as I think this is closest to the existing behaviour, or perhaps 0.001.

You should remove the print statement.

 `if isinstance(percentile, float) or isinstance((percentile, int)):` might be better using `np.isscalar`.

Reply:
> Cool! I would default to 3 sigma, as I think this is closest to the existing behaviour, or perhaps 0.001.

I think 5 sigma quite consistently gives prettier default results. Here a comparison of 3 sigma (top) and 5 sigma (bottom):

![better_1d_3sigma](https://user-images.githubusercontent.com/18258042/85552530-fc81ee80-b61a-11ea-86e1-33814b9de48e.png)

![better_1d_5sigma](https://user-images.githubusercontent.com/18258042/85552551-00ae0c00-b61b-11ea-8275-aa0dd7d9c5bf.png)

The tails nicely hit the floor in the 5 sigma case without extending too far out, but your call.